### DISCUSSION TITLE
How does ImageMagick use GPU hardware acceleration

### DISCUSSION THREAD

**Discussion by zxc155226522** [ImageMagick/ImageMagick#6530] [Help]
### ImageMagick version

7.1.1-6

### Operating system, version and so on

win10

### Description

![image](https://github.com/ImageMagick/ImageMagick/assets/129932013/8e65e28d-fb7b-450d-92c5-912ebd38431d)
![image](https://github.com/ImageMagick/ImageMagick/assets/129932013/d5889bcf-4e5c-464b-a10e-ecebdde0a8bd)

**Comment by dlemstra** [comment]
What is your question? And please post text instead of images.

**Reply by zxc155226522** [reply]
How to use a graphics card to speed up the conversion of pictures

**Reply by dlemstra** [reply]
Some operations will use your graphics card when it is possible. This will be done automatically but you have an option to disable the use of the GPU. And for NVIDIA cards we had to disable OpenCL and we cannot enable that until we get one of those cards that can reproduce the issue.

**Reply by zxc155226522** [reply]
All right. Thank you.

**Comment by Radjin-64** [comment]
Using the M4 pro Mac. What do I need to do to get image magic to utilize the GPU in photo manipulation? Specifically using Piwigo software?
